packages/app/src/breadcrumbs/CopyableCrumb.tsx

```diff
@@ -23,7 +23,7 @@ function CopyableCrumb(props: Props) {
         void navigator.clipboard.writeText(path);
         setPathCopied(true);
       }}
-      onPointerOut={() => setPathCopied(false)}
+      onPointerLeave={() => setPathCopied(false)}
```

Comment:
Bugfix: after copying the path, the tick icon would revert back to the copy icon when moving the mouse from the label to the icon.